### ISSUE TITLE
About scheduler and fibers in Dyo

### ISSUE THREAD

**Issue by mcjazzyfunky** [dyo/dyo#131]
@thysultan Again a question: All that scheduler and fiber rendering stuff in Dyo - is it fully comparable to the upcoming "concurrent mode" in React? Will it also allow (at least in theory) future "concurrent mode" features like the ones listed here?

https://reactjs.org/docs/concurrent-mode-adoption.html#feature-comparison

**Comment by thysultan** [comment]
Yes. current suspense is already supported through this.

**Comment by mcjazzyfunky** [comment]
This is such a killer feature...
You really have to mention this prominently on the dyo.js.org website.
Dyo is really a amazing piece of software.

Thanks a lot for the info.

**Comment by yisar** [comment]
To be honest, I think generator and fiber are two different ways to implement coroutines, and they may not be approximately equal to.
In layman's terms, generator is only the scheduling of js language, and fiber is the browser's coroutine.
